src/core/localization/gettext/ko/common.po

```diff
@@ -854,30 +854,6 @@ msgstr "영국식 단위(베타)"
 msgid "Settings"
 msgstr "설정"
 
-#: profile##englishLanguageOption
-msgid "English"
-msgstr "영어"
-
-#: profile##spanishLanguageOption
-msgid "Spanish"
-msgstr "스페인어"
-
-#: profile##arabicLanguageOption
-msgid "Arabic"
-msgstr "아랍어"
-
-#: profile##koreanLanguageOption
-msgid "Korean"
-msgstr ""
```

Comment:
Is it an option to write each language name as is, except for the file target language?

Reply:
because of i18n:gettext-sync, we sync keys between po files
so if you have 

#: profile##spanishLanguageOption
msgid "Spanish"
msgstr "Español"

in spanish .po files, you need to have it in all other files too

Reply:
Yes, we can have the same translations like this in all `po` files, will it work?

Reply:
hm, i didn't think about it, it will 
so idea is to have 6 repeating keys in all files on different languages
it's also confusing for translator for sure, but maybe better
maybe better to ask for other devs opinion on frontend channel

Reply:
reimplemented on your approach  🚀